### ISSUE TITLE
geom_new of imported generators

### ISSUE THREAD

**Issue by nesnoj** [openego/dingo#82]
Many datasets of renewable generators do not contain geodata in column `geom_new`. This is because methods for DES allocation are not fully implemented yet by @Ludee.

E.g. check with sql:
```
SELECT *
  FROM model_draft.ego_dea_allocation
  WHERE subst_id=489 AND voltage_level='05 (MS)' AND geom_new IS NULL;
```

There's a [workaround](https://github.com/openego/dingo/blob/a9534be04a18b483e4bfc24592ad8f55d972c9c9/dingo/core/__init__.py#L618) implemented to use `geom` where `geom_new` is empty.

**Comment by nesnoj** [comment]
Grid's method `graph_draw()` works now again, fixed in 0f18e30 (did not respect objects GeneratorDingo and CircuitBreakerDingo before).

**Comment by nesnoj** [comment]
Some methods remain open, disregarded genos are allocated randomly with predefined seed. Bugs are fixed.
